### PR TITLE
infuraApiKey -> infuraProjectId

### PR DESCRIPTION
**Pull request by tyler17**
Infura is supposed to stop supporting requests not using the new Dashboard Project Ids today, per: https://blog.infura.io/infura-dashboard-update-9f02d0643eb3

This PR uses the new endpoints in the kovan and mainnet presets, which are used in the integration tests.  An error is thrown if these presets are used without providing an `infuraProjectId`.

### CODE REVIEW

File: test/infura.spec.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import Maker from '../src/index';
 
-test('use Kovan via Infura without API key', async () => {
-  const maker = await Maker.create('kovan', { log: false });
-  await maker.authenticate();
-});
+test('use Kovan via Infura without project id throws', async () => {
+  try {
+  	const maker = await Maker.create('kovan', { log: false });
```
Comment (levity) [inline test/infura.spec.js]: indentation is off -- maybe just run prettier on everything

File: test/eth/Web3Service.spec.js [hunk 2]
```diff
@@ -80,7 +81,7 @@ function buildInfuraService(network, privateKey = null) {
       provider: {
         type: ProviderType.INFURA,
         network,
-        infuraApiKey: process.env.INFURA_API_KEY
+        infuraProjectId: infuraProjectId
```
Comment (levity) [inline test/eth/Web3Service.spec.js]: please use the shorthand syntax here and in the other two cases

**Comment by levity** [comment]
Note btw that this changes the Infura behavior significantly enough that we should mention it in the changelog for the release this goes into.

**Comment by tyler17** [comment]
Makes sense.  I made these style changes and merged it into dev.

**Pull request merged**